src/proto/grpc/auth/v1/BUILD

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+licenses(["notice"])  # Apache v2
+
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+proto_library(
+    name = "sdk_auth_policy_proto",
+    srcs = [
+        "sdk_auth_policy.proto",
```

Comment:
I would name this proto file: sdk_authz_policy.proto
Please also change the corresponding upb files.